com.geeks3d.furmark.yml

```diff
@@ -0,0 +1,51 @@
+id: com.geeks3d.furmark
+
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+
+command: FurMark_GUI
+
+modules:
+
+  - name: libglu
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.freedesktop.org/mesa/glu
+        branch: master
+
+  - name: furmark
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - cp -r furmark /app/furmark/
+      - chmod +x /app/furmark/FurMark_GUI
+      - ln -s /app/furmark/FurMark_GUI /app/bin/FurMark_GUI
+      - ffmpeg -i furmark-icon.png -vf scale=512:512 furmark-icon-scaled.png
+      - install -Dm644 furmark-icon-scaled.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png # Install the app icon
+      - install -Dm644 com.geeks3d.furmark.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop # Install the app menu entry .desktop file
+      - install -Dm644 com.geeks3d.furmark.metainfo.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.metainfo.xml # Install the app metadata
+
+    sources:
+      - type: archive
+        url: https://gpumagick.com/downloads/files/2024/furmark2/FurMark_2.3.0.0_linux64.zip
```

Comment:
This is x86_64 only, you are building for both arches. See https://docs.flathub.org/docs/for-app-authors/requirements#flathubjson